server/lib/directory.coffee

```diff
@@ -0,0 +1,30 @@
+path = require 'path'
+fs = require 'fs'
+root = '/usr/local/var/cozy'
```

Comment:
Is it a standard path? Or a default one? In the second case, an environment variable to override it is welcome.